scripts/upgrade

```diff
@@ -18,10 +18,18 @@ app=$YNH_APP_INSTANCE_NAME
 
 domain=$(ynh_app_setting_get --app=$app --key=domain)
 path_url=$(ynh_app_setting_get --app=$app --key=path)
-port=$(ynh_app_setting_get --app=$app --key=port) #TODO: update port for 8095 one
+port=$(ynh_app_setting_get --app=$app --key=port)
 port_wg=$(ynh_app_setting_get --app=$app --key=port_wg)
 final_path=$(ynh_app_setting_get --app=$app --key=final_path)
 
+# Change port for WireGuard UI because 8095 is default port for WireGuard VPN
+if [ $port -eq 8095 ]
+then
+	ynh_app_setting_delete --app=$app --key=port
+	port=$(ynh_find_port --port=8096)
+	ynh_app_setting_set --app=$app --key=port --value=$port
+fi
+
```

Comment:
It is only cosmetic, but can you put this code under a dedicated section?

```
#=================================================
# ENSURE DOWNWARD COMPATIBILITY
#=================================================
ynh_script_progression --message="Ensuring downward compatibility..." --time --weight=1
...
```

Reply:
I will do it in the next commit